middlewares/RatingsMiddleware.js

```diff
@@ -0,0 +1,32 @@
+import StatusResponse from '../helpers/StatusResponse';
+
+// This function checks for an article id being an integer on [articles rating]
+const checkArticlesId = (req, res, next) => {
+  const { identifier } = req.params;
+  if (!Number.isInteger(+identifier)) {
+    res.status(400).json({
+      errors: {
+        body: ['Articles Id must be an integer']
+      }
+    });
+  }
+  return next();
+};
+
+// This function checks for a user entering valid ratings input [articles rating]
+const validRatingsInput = (req, res, next) => {
+  if (!req.body.stars || req.body.stars < 1 || req.body.stars > 6) {
```

Comment:
I thought our stars max is 5. Review this condition again `req.body.stars > 6`

Reply:
Noted @Chidi3b3r3, Thanks